packages/service-library/src/servicelib/docker_compose.py

```diff
@@ -0,0 +1,40 @@
+import yaml
+
+
+# Notes on below env var names:
+# - SIMCORE_REGISTRY will be replaced by the url of the simcore docker registry
+# deployed inside the platform
+# - SERVICE_VERSION will be replaced by the version of the service
+# to which this compos spec is attached
+# Example usage in docker compose:
+#   image: ${SIMCORE_REGISTRY}/${DOCKER_IMAGE_NAME}-dynamic-sidecar-compose-spec:${SERVICE_VERSION}
+MATCH_SERVICE_VERSION = "${SERVICE_VERSION}"
+MATCH_SIMCORE_REGISTRY = "${SIMCORE_REGISTRY}"
+MATCH_IMAGE_START = f"{MATCH_SIMCORE_REGISTRY}/"
+MATCH_IMAGE_END = f":{MATCH_SERVICE_VERSION}"
+
+
+def replace_env_vars_in_compose_spec(
+    service_spec: "ComposeSpecLabel",
```

Comment:
why is it done like this here? could this not go with the model or as a utils? why is it in the servicelib?

Reply:
Because this is required both by the `catalog` and by the `dynamic-sidecar`.

Reply:
yes, but why not move this to the models_library then? I don't get what is the value having these appart.

Reply:
Because of the following reasons:
- I need to share some code between two packages/services. So I put it into `servicelib`
- the `pyyaml` is required and is not available in `models-library`, who's dependencies we want to keep fixed.